test/hotspot/jtreg/serviceability/attach/RemovingUnixDomainSocketTest.java

```diff
@@ -66,7 +67,7 @@ private static void runJCmd(long pid) throws InterruptedException, IOException {
             "jcmd  exitValue = " + out.getExitValue());
 
         out.shouldHaveExitValue(0);
-        out.stderrShouldBeEmptyIgnoreVMWarnings();
+        out.stderrShouldBeEmptyIgnoreDeprecatedWarnings();
```

Comment:
Looked at the fix one more time.

I suppose this new method is not needed as a the message about deprecation is VM warning and should be handled by stderrShouldBeEmptyIgnoreVMWarnings.
Also this new method will fails if jcmd prints some other VM warning